src/scifem.cpp

```diff
@@ -10,11 +10,46 @@
 #include <dolfinx/mesh/Mesh.h>
 #include <memory>
 #include <nanobind/nanobind.h>
+#include <nanobind/ndarray.h>
 #include <nanobind/stl/shared_ptr.h>
 #include <nanobind/stl/tuple.h>
 #include <nanobind/stl/vector.h>
 
-using namespace dolfinx;
+namespace
+{
+// Copyright (c) 2020-2024 Chris Richardson, Matthew Scroggs and Garth N. Wells
+// FEniCS Project (Basix)
+// SPDX-License-Identifier:    MIT
```

Comment:
Is some of this copied from Basix? If so, it is not possible to include this in the header?

Reply:
Only this namespace;)